docs/en/ingest-management/elastic-agent/install-fleet-managed-elastic-agent.asciidoc

```diff
@@ -78,6 +78,15 @@ To install an {agent} and enroll it in {fleet}:
 . In the *Add agent* flyout, select an existing agent policy or create a new
 one. If you create a new policy, {fleet} generates a new
 {fleet-guide}/fleet-enrollment-tokens.html[{fleet} enrollment token].
++
+[NOTE]
+====
+For on-premises deployments, you can dedicate a policy to all the
+agents in the network boundary and configure that policy to include a
+specific {fleet-server} (or a cluster of {fleet-server}s).
+
+Read more in <<add-fleet-server-to-policy>>.
```

Comment:
Try using an external link here because line 73 starts a tagged region.

Reply:
It worked! Thanks @dedemorton!